linera-core/src/chain_worker/state/mod.rs

```diff
@@ -130,7 +130,7 @@ where
     pub(super) async fn read_certificate(
         &mut self,
         height: BlockHeight,
-    ) -> Result<Option<Certificate>, WorkerError> {
+    ) -> Result<Option<linera_chain::data_types::Certificate>, WorkerError> {
```

Comment:
We should double-check, but I believe this will always be a `ConfirmedBlockCertificate` read from storage?